dashboard/server.R

```diff
@@ -12,10 +12,10 @@ source("generate_subnational_map.R")
 server <- function(input, output) {
     
     # create variables for period values
-    fn_ml1 <- ipc_indices_data %>% filter(Source == 'FewsNet') %>% select(ML1_period) %>% unique() %>% as.character()
-    fn_ml2 <- ipc_indices_data %>% filter(Source == 'FewsNet') %>% select(ML2_period) %>% unique() %>% as.character()
-    gbl_ml1 <- ipc_indices_data %>% filter(Source == 'GlobalIPC') %>% select(ML1_period) %>% unique() %>% as.character()
-    gbl_ml2 <- ipc_indices_data %>% filter(Source == 'GlobalIPC') %>% select(ML2_period) %>% unique() %>% as.character()
+    fn_ml1 <- ipc_indices_data_latest %>% filter(source == 'FewsNet') %>% select(period_ML1) %>% unique() %>% as.character()
+    fn_ml2 <- ipc_indices_data_latest %>% filter(source == 'FewsNet') %>% select(period_ML2) %>% unique() %>% as.character()
+    gbl_ml1 <- ipc_indices_data_latest %>% filter(source == 'GlobalIPC') %>% select(period_ML1) %>% unique() %>% as.character()
+    gbl_ml2 <- ipc_indices_data_latest %>% filter(source == 'GlobalIPC') %>% select(period_ML2) %>% unique() %>% as.character()
 
     # conditionally select correct map
     output$trigger_map <- renderPlot({
```

Comment:
@Tinkaa Here's why your radio buttons don't work. You removed the "country" variable from the csv.